apps/dashboard/src/components/icons/utils.ts

```
@@ -10,16 +10,16 @@ import { Mail3Fill } from './mail-3-fill';
 import { Notification5Fill } from './notification-5-fill';
 import { Sms } from './sms';
 import { IconType } from 'react-icons/lib';
-import { StepTypeEnum } from '@/utils/enums';
+import { StepTypeEnum } from '@novu/api/models/components';
```

ESLint

[error] `@novu/api/models/components` import should occur before import of `./mail-3-fill`

(import/order)